Gemfile

```diff
@@ -22,10 +22,10 @@ gem 'devise-encryptable', '0.1.1'
 gem 'devise_security_extension', '0.7.2', git: "https://github.com/alphagov/devise_security_extension.git", branch: "graceful_return_to_behaviour"
 gem 'devise_zxcvbn', '1.1.1'
 gem 'devise-async', '0.8.0'
+gem 'pundit'
```

Comment:
I'm strongly in favour of pinning gem versions to document intent and to make it much easier and clearer to track changes over time.